locales/en/inferenceai.json

```diff
@@ -0,0 +1,84 @@
+{
+  "hero": {
+    "heading": "Build Custom AI Agents That Actually Get Work Done",
+    "desc": "We help businesses design, develop, and deploy purpose-built AI agents that save time, cut costs, and scale operations—without the technical overwhelm."
+  },
+  "cta": {
+    "heading": "Ready to Transform Your Workflow?",
+    "desc": "Chat with our team to discover how AI can streamline your business.",
+    "label": "Schedule Consultation"
+  },
+  "navItems": {
```

Comment:
You need to be consistent with the JSON keys. Since the existing code uses snake case, just stick with snake case.